packages/trie/CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 5.0.0-beta.3 - 2022-08-10
+
+Beta 3 release for the upcoming breaking release round on the [EthereumJS monorepo](https://github.com/ethereumjs/ethereumjs-monorepo) libraries, see the Beta 1 release notes for the main long change set description as well as the Beta 2 release notes for notes on some additional changes ([CHANGELOG](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/devp2p/CHANGELOG.md)).
+
+### Root Hash Persistance
+
+The trie library now comes with a new constructor option `persistRoot` which is disabled by default but allows to persist state root updates along write operations directly in the DB and therefore omits the need to manually set to a new state root, see PR [#2071](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2071), thanks to @faustbrian for the contribution! ❤️
+
+To switch back to the old behavior you can pass in the option set to `false` on instantiation:
```

Comment:
We should remove this line now or else switch to the below
```suggestion
To switch to the new behavior you can pass in the option set to `true` on instantiation:
```

Reply:
Same comment, this should be updated to `true` and the example below adjusted accordingly.

Reply:
Sorry for dismissing again, but I just do not have time to have the tests running once again and would want to have these releases still out on Thursday and not on a Friday. 😬

Reply:
Finally fixed this in a doc fixes/improvements PR I will push soon. 🙂